apps/server/src/tasks/tasks.service.ts

```diff
@@ -19,6 +20,79 @@ import { Task } from './entities/task.entity';
 
 @Injectable()
 export class TasksService {
+  @Cron('* */1 * * * *')
+  triggerMarkingAsDone() {
+    this.markTasksAsDone();
+  }
```

Comment:
Can we just mark `markTaskAsDone` as a Cron to not create this layers?

Reply:
Wrote it this way since the markTasksAsDone is an async function and the compiler does not allow to put an `aync` keyword before the cron function signature name

Reply:
Why? How is it supposed to work to not create boilerplate...?
But it's nothing blocking, can be delegated for now